examples/two_case_catke_calibration.jl

```diff
@@ -0,0 +1,225 @@
+          CᴷRiᶜ = Normal(0.2, 0.01))
+
+free_parameters = FreeParameters(priors)
+
+# TODO: write out a table of the parameters with their physical meanings...
+#
+# Finally, we build the `InverseProblem` and `EnsembleKalmanInversion`, and
+# execute one iteration of `EnsembleKalmanInversion` with relatively large noise,
+
+calibration = InverseProblem(observations, simulation, free_parameters)
+
+eki = EnsembleKalmanInversion(calibration;
+                              noise_covariance = 1.0,
+                              resampler = NaNResampler(abort_fraction=0.5))
+
+# We also build some utilities for comparing observed and model output later on:
+
+Nt = length(times)
+observed_data = []
+modeled_data = []
+
+for observation in observations
+    time_serieses = observation.field_time_serieses
+    case_data = NamedTuple(n => getproperty(time_serieses, n)[Nt] for n in field_names)
+    push!(observed_data, case_data)
+end
+
+function get_modeled_case(case, name)
+    model_time_serieses = calibration.time_series_collector.field_time_serieses 
+    field = getproperty(model_time_serieses, name)[Nt]
+    return interior(field)[case, :, :]
+end
+
+for case = 1:2
+    push!(modeled_data, NamedTuple(n => get_modeled_case(case, n) for n in field_names))
+end
+
+function compare_model_observations(model_label="modeled")
+    fig = Figure(resolution=(1200, 800))
+    for case = 1:2
+        axs = make_axes(fig, case)
+        obs = observed_data[case]
+        modeled = modeled_data[case]
+        plot_fields!(axs, obs..., "observed at t = " * prettytime(times[end]), :black, :solid)
+        plot_fields!(axs, modeled..., model_label, :blue, :solid)
+        [axislegend(ax, position=:rb, labelsize=10) for ax in axs]
+    end
+    return fig
+end
+
+# Now let's iterate to see if we can find better parameters than our initial guess...
+
+iterate!(eki; iterations = 5)
+
+# To evaluate how 
+
+initial_parameters = eki.iteration_summaries[0].ensemble_mean
+best_parameters = eki.iteration_summaries[end].ensemble_mean
+Niter = length(eki.iteration_summaries) - 1
+
+forward_run!(calibration, initial_parameters)
+fig = compare_model_observations("modeled after 0 iterations")
+save("two_case_catke_comparison_iteration_0.svg", fig); nothing # hide
+display(fig)
+
+# ![](two_case_catke_comparison_iteration_0.svg", fig)
+
+forward_run!(calibration, best_parameters)
+fig = compare_model_observations("modeled after $Niter iterations")
+save("two_case_catke_comparison_iteration_$Niter.svg", fig); nothing # hide
+display(fig)
+
+# ![](two_case_catke_comparison_iteration_5.svg", fig)
+
+# Let's see how the parameters evolved:
+
+summaries = eki.iteration_summaries
+parameter_names = keys(first(summaries).ensemble_mean)
+
+ensemble_means = NamedTuple(name => map(summary -> summary.ensemble_mean[name], summaries)
+                            for name in parameter_names)
+fig = Figure()
+ax = Axis(fig[1, 1], xlabel = "Iteration", ylabel = "Parameter value")
```

Comment:
Are the parameters diffusivities?

```suggestion
ax = Axis(fig[1, 1], xlabel = "Iteration", ylabel = "Parameter value [m² s⁻¹]")
```

Reply:
No, the parameters are non-dimensional.